src/store/annotations/actions.ts

```diff
@@ -22,4 +28,13 @@ export const createAnnotationAction = createAsyncThunk('CREATE_ANNOTATION', asyn
     return response.data;
 });
 
+export const fetchAnnotationsAction = createAsyncThunk<Annotation[], FetchArg, ThunkAPI>(
+    'FETCH_ANNOTATIONS',
+    async ({ fileId, fileVersionId }, { extra }) => {
```

Comment:
Should `FetchArg` and `ThunkAPI` be typing the arguments to the payloadCreator or are they not necessary?

Reply:
They are typed as part of the `createAsyncThunk<Annotation[], FetchArg, ThunkAPI>` declaration above.